defaults/main.yml

```diff
@@ -3,6 +3,8 @@ grafana_repo_release: stable
 grafana_service_name: grafana-server
 grafana_service_enabled: true
 grafana_version: 5.0.0
+grafana_upstream_pkg_repository: true
```

Comment:
pkg ? package

Reply:
because I'd like to test it manually first